pkgs/development/compilers/intel-graphics-compiler/default.nix

```diff
@@ -1,6 +1,8 @@
-{ lib, stdenv
+{ lib
+, stdenv
 , fetchFromGitHub
 , cmake
+, runCommandLocal
 , pkg-config
```

Comment:
`pkg-config` isn't being used anywhere and can probably be removed now, unless it's actually needed.

Reply:
sure. lets remove it